src/Cassandra.IntegrationTests/SimulacronTest.cs

```diff
@@ -0,0 +1,206 @@
+//
+//       Copyright (C) DataStax Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using Cassandra.IntegrationTests.SimulacronAPI.Models.Logs;
+using Cassandra.IntegrationTests.SimulacronAPI.SystemTables;
+using Cassandra.IntegrationTests.TestClusterManagement;
+using Cassandra.IntegrationTests.TestClusterManagement.Simulacron;
+using NUnit.Framework;
+
+namespace Cassandra.IntegrationTests
+{
+    [TestFixture, Category("short")]
+    public class SimulacronTest
```

Comment:
Good idea to have a clean base class for simulacron-based tests.

Shouldn't this be `abstract`?

Reply:
yep good catch, done